packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -464,21 +464,44 @@ export function TransferPanelMain({
     if (isDepositMode) {
       return {
         from: {
-          disabled: true,
```

Comment:
I think the implementation at the current deployment is fine for mainnet, one & nova
It's only weird for testnet L1 and L2 because `disabled` is hardcoded to be `true` for `from` and `false` for `to` but on testnet both dropdowns only have one option so they should both be `disabled: true`

If we set the `disabled` prop based on the length of the `options` array, it'll fix the weirdness on testnet, but it doesn't matter to our users

and when we support One <-> Nova bridge, we can add some logic for network switching?

Reply:
I will look into disabling it for a single option.